### PR TITLE
create the project models

### PR DESCRIPTION
**Pull request by zampeta**

### CODE REVIEW

**Comment by luminitacristinalazarescu** [comment]
I renamed the project and all the files and I removed flake8 from travis :smile: since the migration files generated will have too long lines and is too much to modify them every time so that travis passes
But still we should keep [this|https://www.python.org/dev/peps/pep-0008/] in mind.

**Comment by mei-li** [comment]
I put the flake8 back in travis, with config to avoid migration files that violate pep8 rules
@zampeta
So flake8 is running for each commit on a Pull Request (PR), it is a code style checker, that controls for pep8 (read what is pep8 [here](https://www.python.org/dev/peps/pep-0008/))
 can you try to fix these issues for practising pep8

**Comment by mei-li** [comment]
so try to fix the issues and make the travis to tick green

**Comment by zampeta** [comment]
ok, i will take a look!

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
```
Comment (luminitacristinalazarescu) [inline setup.cfg]: How is this file used? I don't understand how travis or the project uses it in order to set these rules when checking pep8.

**Comment by mei-li** [comment]
:+1:

**Comment by mei-li** [comment]
Fixes https://github.com/luminitacristinalazarescu/codeeve/issues/4

**Pull request merged**